LISA/LISA.cpp

```diff
@@ -42,22 +42,23 @@ namespace Plugin {
         _service = service;
         _service->Register(&_notification);
 
+        string message;
         _lisa = service->Root<Exchange::ILISA>(_connectionId, 2000, _T("LISAImplementation"));
         if (_lisa != nullptr) {
             Config config;
             config.FromString(service->ConfigLine());
             std::string path = (config.DbPath.IsSet() && !config.DbPath.Value().empty()) ? config.DbPath.Value() : service->PersistentPath();
             _lisa->Configure(path);
 
-            TRACE(Trace::Information, (_T("LISA::Initialize register notification")));
+            TRACE_L1("register notification");
```

Comment:
Shouldn't be INFO(...) ?

Reply:
That's for the Initialization part - we agreed that the logs should go unconditionally here since it's too early to even enable logs for LISA plugin for that code so they would not ever be seen. @szymon-polke-red please confirm

Reply:
yes, same for deinitialization